opentelemetry/src/trace/span_context.rs

```diff
@@ -20,6 +20,14 @@ use thiserror::Error;
 pub struct TraceFlags(u8);
 
 impl TraceFlags {
+    /// Trace flags with the `sampled` flag set to `0`.
```

Comment:
Style suggestion: I usually like to put `const`s near the bottom instead of near the top, since they're usually relatively simple and don't change often.